src/ops.rs

```diff
@@ -200,6 +201,62 @@ operator!(impl<('a)(A, B)> for &'a BitSetOr<A, B>);
 operator!(impl<()(A, B)> for BitSetXor<A, B>);
 operator!(impl<('a)(A, B)> for &'a BitSetXor<A, B>);
 
+macro_rules! iterator {
+    ( $index:ty, $bitset:ident ) => {
+        impl FromIterator<$index> for $bitset {
```

Comment:
Why is this macro generic over the index?

Reply:
Remnants from when I thought it would be easier to specify &'a Index and Index for the traits.